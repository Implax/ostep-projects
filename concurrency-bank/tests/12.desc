Correctness test: single negative transaction
