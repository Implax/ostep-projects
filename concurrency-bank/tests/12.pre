ln -sf $(pwd)/tests/assets/no_transactions.txt /tmp/Husband.txt && ln -sf $(pwd)/tests/assets/negative_transaction.txt /tmp/Wife.txt
