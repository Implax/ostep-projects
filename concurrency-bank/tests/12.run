./bank 400 tests/assets/no_transactions.txt tests/assets/negative_transaction.txt