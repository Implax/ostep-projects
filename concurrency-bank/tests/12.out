Opening balance: 400
Withdraw: -40, User: Wife, Transaction declined
Closing balance: 400
